.github/workflows/release.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches: [master]
     tags: ["*"]
+  release:
+    types: [published]
```

Comment:
Is this line necessary? I though the "published" step already creates a tag that triggers this job 🤔

Reply:
That's always what I'd have thought, but it appears that this step was not triggered for 1.2.0-M1 which created through GH releases.

Reply:
Interesting, let's try this then 👍